job/server/src/main/java/alluxio/master/job/tracker/CmdJobTracker.java

```diff
@@ -270,4 +289,29 @@ public CmdStatusBlock getCmdStatusBlock(long jobControlId)
             .collect(Collectors.toList());
     return new CmdStatusBlock(cmdInfo.getJobControlId(), blockList, cmdInfo.getOperationType());
   }
+
+  private void cleanUnableTracedJobs() {
+    long currentTime = System.currentTimeMillis();
+    for (Map.Entry<Long, CmdInfo> x : mInfoMap.entrySet()) {
+      CmdInfo cmdInfo = x.getValue();
+      if (currentTime - cmdInfo.getJobSubmissionTime() > sTraceRetentionTime) {
```

Comment:
if (currentTime - cmdInfo.getJobSubmissionTime() > sTraceRetentionTime)

(currentTime - cmdInfo.getJobFinishedTime() - sTraceRetentionTime = 1s

Users will never be able to query the results of this load？

Reply:
[TODO]